cdap-examples/Sports/src/main/java/co/cask/cdap/examples/sports/SportsResults.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.examples.sports;
+
+import co.cask.cdap.api.app.AbstractApplication;
+import co.cask.cdap.api.dataset.lib.PartitionedFileSet;
+import co.cask.cdap.api.dataset.lib.PartitionedFileSetProperties;
+import co.cask.cdap.api.dataset.lib.Partitioning;
+import org.apache.hadoop.mapreduce.lib.input.TextInputFormat;
+import org.apache.hadoop.mapreduce.lib.output.TextOutputFormat;
+
+/**
+ * An example that illustrates partitioned file sets at the example of sport result analytics.
+ */
+public class SportsResults extends AbstractApplication {
+
+  @Override
+  public void configure() {
+    addService(new UploadService());
+    addMapReduce(new ScoreCounter());
+
+    // create the results partitioned file set, configure it to work with MapReduce and with Explore
+    createDataset("results", PartitionedFileSet.class, PartitionedFileSetProperties.builder()
+      // properties for partitioning
+      .setPartitioning(Partitioning.builder().addStringField("league").addIntField("season").build())
+        // properties for file set
+      .setInputFormat(TextInputFormat.class)
+      .setOutputFormat(TextOutputFormat.class)
+      .setInputProperty(TextOutputFormat.SEPERATOR, ",")
+        // properties for explore (to create a partitioned hive table)
```

Comment:
Do you want to go through all the other examples and change all comments to upper case?